aws-lc-rs/src/cipher/streaming.rs

```diff
@@ -0,0 +1,527 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+use crate::cipher::{
+    Algorithm, DecryptionContext, EncryptionContext, OperatingMode, UnboundCipherKey,
+};
+use crate::error::Unspecified;
+use crate::ptr::{LcPtr, Pointer};
+use aws_lc::{
+    EVP_CIPHER_CTX_new, EVP_CIPHER_iv_length, EVP_CIPHER_key_length, EVP_DecryptFinal_ex,
+    EVP_DecryptInit_ex, EVP_DecryptUpdate, EVP_EncryptFinal_ex, EVP_EncryptInit_ex,
+    EVP_EncryptUpdate, EVP_CIPHER_CTX,
+};
+use std::ptr::null_mut;
+
+/// A cipher encryption key for streaming encryption operations.
+pub struct StreamingEncryptingKey {
```

Comment:
Generally you don't need to prefix the struct with the module name (see https://rust-lang.github.io/rust-clippy/master/#/module_name_repetitions)
```suggestion
pub struct EncryptingKey {
```